site/content/getting-started/_index.md

```diff
@@ -4,7 +4,7 @@ description: Getting Started
 id: getting-started
 ---
 
-This page will help you get up and running with Contour.
+This page will help you get up and running with Project Contour.
```

Comment:
I don't feel strongly about the "Project" prefix in Contour. I always liked just "Contour". 

Project was added since someone else already had the domain name. But that said, happy to merge. =)

Reply:
was just a demo PR for a video :) didn't have the time to close it :)